From 2df864dea2e680d06c7011dc2e81a078f113529b Mon Sep 17 00:00:00 2001
From: mihai lindner <mihaix.lindner@intel.com>
Date: Wed, 25 Apr 2012 15:22:31 +0300
Subject: check for existing $JAVAC before setting it

for linux environments that have $JAVAC already defined by the system
env, when JDK is present.

Change-Id: I941874bdeba3c1818fb738af9d3e3da22c0a67f3
---
 core/find-jdk-tools-jar.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/core/find-jdk-tools-jar.sh b/core/find-jdk-tools-jar.sh
index f150a9a..6c73c5a 100755
--- a/core/find-jdk-tools-jar.sh
+++ b/core/find-jdk-tools-jar.sh
@@ -2,7 +2,7 @@
 if [ "x$ANDROID_JAVA_HOME" != x ] && [ -e "$ANDROID_JAVA_HOME/lib/tools.jar" ] ; then
     echo $ANDROID_JAVA_HOME/lib/tools.jar
 else
-    JAVAC=$(which javac)
+    [[ -x $JAVAC ]] || JAVAC=$(which javac)
     if [ -z "$JAVAC" ] ; then
         exit 1
     fi
-- 
1.7.4.1

